WriteVHDLSyntax.py

```diff
@@ -202,6 +202,8 @@ def writeTBMemoryReadInstance(mtypeB, memDict, bxbitwidth, is_initial, is_binned
                 memtmp = memtmp.replace("n1","")
             #    memtmp = "T"+mem[1:10]
             string_mem += "      FILE_NAME".ljust(str_len) + "=> FILE_IN_" + mtypeB+"&\""+ memtmp + "\"&inputFileNameEnding,\n"
+            if "MPAR" in mem:
+                string_mem += "      PAGE_LENGTH".ljust(str_len) + "=> 128,\n"
```

Comment:
@aryd Since the default page length for `tf_mem_tpar` is already defined to be 128 (https://github.com/cms-L1TK/firmware-hls/blob/173f76299fd34867e084a3337c819216c99fedb4/IntegrationTests/common/hdl/tf_pkg.vhd#L42), I would prefer to omit this parameter to avoid having a magic 128 in this script:
```suggestion
```

Reply:
Fixed.